sections/annoyances.txt

```diff
@@ -19,6 +19,13 @@ hu###cookie-bar
 hu###cookie-notice
 hu###cookie-window
 hu###cookiebar
+hu##.cookie
```

Comment:
_maybe_ this rule is too general and can cause issues

Reply:
i just checked, none of the popular cookie filters contain such a rule, probably to avoid many possible issues